### DISCUSSION TITLE
posts/2021-06-02_color-scheme-anc-states/

### DISCUSSION THREAD

**Discussion by giscus** [joelnitta/joelnitta-home#5] [Comments]
# posts/2021-06-02_color-scheme-anc-states/

How to change the phytools default color scheme when visualizing the results of ancestral character state estimation

https://www.joelnitta.com/posts/2021-06-02_color-scheme-anc-states/

**Comment by joelnitta** [comment]
testing testing 123

**Reply by shannonpileggi** [reply]
it works!
